src/utils/helpers.ts

```diff
@@ -72,6 +72,18 @@ export const isBrowser = typeof window !== 'undefined' && typeof window.document
 
 export const noop = (): void => {};
 
+export const isEmpty = (value?: string | number | readonly string[] | undefined): boolean => {
```

Comment:
to avoid lodash clashes (and possibly developer mistakes), can we rename this to `isEmptyValue`?

If we do that, perhaps the comment below about arrays and objects might not be needed

Reply:
That was my initial thought, your suggestion makes totally sense.
Fixed!